packages/strapi-design-system/test-bundler.js

```diff
@@ -1,42 +1,35 @@
 import IntlPolyfill from 'intl';
+import ResizeObserver from 'resize-observer-polyfill';
```

Comment:
How come this one and not https://github.com/juggle/resize-observer ? I think the latter is actually maintained regularly.

Reply:
I don't mind which one to choose. If you think this one works better, I'm ok with that 👍🏼

Reply:
I just looked at when they were last updated 😅